app/src/main/java/com/example/easycloset/MainActivity.java

```diff
@@ -89,10 +90,27 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         return true;
     }
 
-    private void setFragmentContainer(Fragment fragment) {
+    public ClosetFragment getClosetFragment() {
+        return closetFragment;
+    }
+
+    public ProfileFragment getProfileFragment() {
+        return profileFragment;
+    }
+
+    public SuggestFragment getSuggestFragment() {
```

Comment:
why all of the methods public?

Reply:
so i can change the fragment when clicked on